tests/test-steering-render.sh

```
@@ -534,6 +534,28 @@ rm -f "$HVER/.gemini/GEMINI.md"
+  pass "command-line paths in routed documents resolve to the installed copies"
+else
+  fail "local-review.md still names principles/local-review-contract.md as a bare path: $(grep -n 'local-review-contract.md' "$HCMD/.touchstone/principles/local-review.md" | head -2 | tr '\n' ' ')"
+fi
+
+# A home containing a single quote cannot be rendered into a pasteable
+# single-quoted command, so install refuses it by its canonical path.
+HQ="$TMP_DIR/it's home"
+if bash "$INSTALL" install --home "$HQ" >"$TMP_DIR/hq.out" 2>&1; then
+  fail "install accepted a home containing a single quote"
+else
+  grep -q "contains a single quote" "$TMP_DIR/hq.out" && pass "a home with a single quote is refused by name" || fail "unexpected refusal: $(cat "$TMP_DIR/hq.out")"
```

Shellcheck (0.11.0)

[info] Note that A && B || C is not if-then-else. C may run when A is true.

(SC2015)